xwiki-platform-core/xwiki-platform-menu/xwiki-platform-menu-test/xwiki-platform-menu-test-tests/src/test/it/org/xwiki/menu/test/ui/MenuInAdministrationTestIT.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.menu.test.ui;
+
+import java.util.Arrays;
+
+import org.junit.Test;
+import org.xwiki.administration.test.po.AdministrationPage;
+import org.xwiki.appwithinminutes.test.po.EntryNamePane;
+import org.xwiki.menu.test.po.MenuEntryEditPage;
+import org.xwiki.menu.test.po.MenuHomePage;
+import org.xwiki.model.reference.DocumentReference;
+import org.xwiki.test.ui.AbstractTest;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Test to check that the Menu app is available through the administration and can be used to add new entries
+ *
+ */
+public class MenuInAdministrationTestIT extends AbstractTest
+{
+    @Test
+    public void verifyMenuIsAvailableInAdministration()
+    {
+        // Log in as superadmin
+        getUtil().login("superadmin", "pass");
+
+        DocumentReference menu1Reference = new DocumentReference("xwiki", Arrays.asList("Menu", "menu1"), "WebHome");
+        getUtil().deletePage(menu1Reference);
```

Comment:
If this is just setup and the point is not really to test the delete UI you should use

    getUtil().rest().deletePage(menu1Reference)

which is much faster.

Reply:
` getUtil().deletePage(menu1Reference);` is meant to NOT navigate too (it's a setup method).

Reply:
getUtil().rest().delete(menu1Reference) and getUtil().deletePage() should both do exactly the same thing.

Reply:
Note: The nav one is very different, it's supposed to be available from ViewPage by clicking on menus, etc.

Reply:
BTW I don't like that we made the API more complex than before. As a user of the API I don't need to know I'm using REST and `getUtil().deletePage()` is a much nicer api than `getUtil().rest().delete()`.